cmd/samples/cron/main.go

```diff
@@ -1,66 +1,108 @@
 package main
 
 import (
+	"context"
 	"flag"
+	"os"
+	"os/signal"
 	"time"
 
 	"github.com/pborman/uuid"
 	"go.temporal.io/temporal/client"
 	"go.temporal.io/temporal/worker"
-
-	"github.com/temporalio/temporal-go-samples/cmd/samples/common"
+	"go.uber.org/zap"
 )
 
 const (
-	// ApplicationName is the task list for this sample
-	ApplicationName = "cronGroup"
+	// TaskListName is the task list for this sample.
+	TaskListName = "cron-task-list"
+)
+
+var (
+	logger *zap.Logger
 )
 
+func main() {
+	var err error
+	logger, err = zap.NewDevelopment()
+	if err != nil {
+		panic(err)
+	}
+
+	var mode string
+	var cron string
+	flag.StringVar(&mode, "m", "", "Mode is worker or trigger.")
+	flag.StringVar(&cron, "cron", "* * * * *", "Crontab schedule.")
+	flag.Parse()
+
+	switch mode {
+	case "worker":
+		w := startWorker()
+		// The workers are supposed to be long running process that should not exit.
+		// Use channel to wait for Ctrl+C.
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt)
+		<-c
+		// Stop worker, close connection, clean up resources.
+		w.Stop()
```

Comment:
Added two extra lines (and some complexity) but I believe it is good to show that workers must/can be stopped.